nestjs-prisma/libs/prisma-schema-one/prisma/schema.prisma

```diff
@@ -0,0 +1,31 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+  output   = "../../../node_modules/@prisma/client/one"
```

Comment:
Should we add a comment explaining why `output` is set the way it is? i.e. so users can import `@prisma/client/one` rather than the default `@prisma/client` which doesn't work well for monorepos.